Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -71,12 +64,18 @@ def download_data(cls, data_dir):
 
         '''
         if os.path.isdir(os.path.join(data_dir, cls.FOLDER)):
-            print("Dataset already exists in " + data_dir)
+            print("Dataset folder already exists in " + data_dir)
         else:
-            if input("Are you sure you want to download " + cls.FILE_SIZE + " dataset from " + cls.URL + " ? (y/n)") == "y":
-                print('Downloading dataset from ' + cls.URL)
-                cls._download_and_extract_from_url(os.path.join(data_dir,cls.FOLDER))
-                print('Download complete')
+            # get user confirmation for download
+            user_input = 'y' if not prompt else input("Are you sure you want to download {} dataset from Zenodo record {} ? (y/n)").format(cls.ZIP_FILE, cls.ZENODO_RECORD)
+            if  user_input == "y": 
+                zenodo_get([cls.ZENODO_RECORD, '-g', cls.ZIP_FILE, '-o', data_dir])
+
+                # unzip file
+                with ZipFile(os.path.join(data_dir, cls.ZIP_FILE), 'r') as zip_ref:
+                    zip_ref.extractall(os.path.join(data_dir, cls.FOLDER))
+                os.remove(os.path.join(data_dir, cls.ZIP_FILE))
+
             else:
                 print('Download cancelled')
```

Comment:
```suggestion
            return True
            else:
                print('Download cancelled')
                return False
```

might be useful to return success as a `bool` for end-user scripts...